app/models/PropertyType.scala

```diff
@@ -49,6 +53,7 @@ object PropertyType {
       case Rentals          => "Rentals"
       case RentARoom        => "RentARoom"
       case RentalsRentARoom => "RentalsRentARoom"
+      case UKProperty       => "uk-property"
```

Comment:
```suggestion
      case UKProperty       => "UKProperty"
```
Camel case seems to be more consistent with the rest